br/pkg/lightning/log/log.go

```diff
@@ -97,7 +97,7 @@ func InitLogger(cfg *Config, _ string) error {
 	// Disable annoying TiDB Log.
 	// TODO: some error logs outputs randomly, we need to fix them in TiDB.
```

Comment:
When do we complete this TODO?

Reply:
I'm not sure, I don't know the details about this problem